server/world/world.go

```diff
@@ -1581,22 +1581,22 @@ func (w *World) setRaining(raining bool) {
 	}
 
 	if raining {
-		w.setRainTime(rand.Intn(12000) + 12000)
+		w.setRainTime(int(x.Seconds() / 20)) // rand.Intn(12000) + 12000
```

Comment:
I think this is supposed to be * 20. You could also do `x / (time.Second*20)` to avoid floats.

Reply:
Can remove the comment here too.